### PR TITLE
cleanup(js): move assets to js package

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

Assets utils currently lives in workspace

## Expected Behavior

Assets utils should live in js

## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by mandarini — APPROVED** [review]

**Review by nartc — APPROVED** [review]

File: packages/js/package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "babel-plugin-macros": "^2.8.0",
     "babel-plugin-transform-typescript-metadata": "^0.3.1",
     "chalk": "^4.1.0",
+    "glob": "7.1.4",
     "fast-glob": "3.2.7",
```
Comment (AgentEnder) [inline packages/js/package.json:50]: Would now be a good time to sawp these from glob -> fast-glob? I know this isn't a particularly perf sensitive codepath, but having both installed under js looks a bit silly.

**Review by AgentEnder — CHANGES_REQUESTED** [review]

File: packages/js/package.json (hunk 1 above)
Comment (AgentEnder) [reply packages/js/package.json:50]: The imports are part of the file rename

Comment (Coly010) [reply packages/js/package.json:50]: Perhaps not now as this Pr is mainly just to move the assets utils from workspace, a further PR can be made to focus on improving the perf

Comment (Coly010) [reply packages/js/package.json:50]: Unless the APIs are similar, in which I could try?

Comment (Coly010) [reply packages/js/package.json:50]: The assets file didnt have associated test files, so where could I confirm that it is functioning correctly if i was to refactor it?

Comment (AgentEnder) [reply packages/js/package.json:50]: The APIs are identical IIRC, I'm not sure what all uses these utils still though, you'd have to look for usages and check manually (or write the tests)

Comment (Coly010) [reply packages/js/package.json:50]: swc + tsc in the js package i believe. I'll check and make the change

Comment (AgentEnder) [reply packages/js/package.json]: swc + tsc use the copyAssetsHandler that's already in the JS package

Comment (Coly010) [reply packages/js/package.json]: Yeah but they also use what's in workspace: https://github.com/nrwl/nx/blob/ebdb315a3a416e4ba44a0b8b69462243a3d71094/packages/js/src/executors/swc/swc.impl.ts#L56

I've made the change anyway for those utils to use fast-glob.

Comment (Coly010) [reply packages/js/package.json]: @AgentEnder can you double check that this is ok?

Comment (AgentEnder) [reply packages/js/package.json]: Yeah - I'll look at it now :)

**Review by AgentEnder — APPROVED** [review]

**Pull request merged**